src/callbacks_stage/callbacks_stage.jl

```diff
@@ -3,9 +3,6 @@
 # we need to opt-in explicitly.
 # See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
 @muladd begin
+#! format: noindent
 
-
-include("positivity_zhang_shu.jl")
-
-
-end # @muladd
+include("positivity_zhang_shu.jl") end # @muladd
```

Comment:
Should be fixed - maybe just remove the `@muladd` block here completely?

Reply:
👍

Reply:
Did you run this with Julia v1.8 or v1.9? With the current JuliaFormatter@1.0.32 I get the following file:
```julia
# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
# Since these FMAs can increase the performance of many numerical algorithms,
# we need to opt-in explicitly.
# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
@muladd begin
#! format: noindent

include("positivity_zhang_shu.jl")
end # @muladd
```

Reply:
Julia v1.9